src/app/components/mnoe-staffs-list/mnoe-staffs-list.coffee

```diff
@@ -76,6 +76,20 @@
             toastr.error('mnoe_admin_panel.dashboard.staffs.widget.list.toastr_error')
         )
 
+      update: (staff)->
+        # Ask for confirmation if the user is updated to admin or division admin as their clients will be cleared
+        if staff.admin_role_was == 'staff' && staff.admin_role != 'staff'
+          modalOptions =
+            closeButtonText: 'mnoe_admin_panel.dashboard.staff.update_staff_role.cancel'
+            actionButtonText: 'mnoe_admin_panel.dashboard.staff.update_staff_role.action'
+            headerText: 'mnoe_admin_panel.dashboard.staff.update_staff_role.proceed'
+            bodyText: 'mnoe_admin_panel.dashboard.staff.update_staff_role.perform'
+          MnoConfirm.showModal(modalOptions).then(
+            => @_updateStaffAction(staff),
```

Comment:
Was this intentional? It works though, ES6 ftw 👍

Reply:
It was :)
https://coffeescript.org/#fat-arrow

So that I can use `@` and not `vm.staff`